components/renku_data_services/notebooks/blueprints.py

```diff
@@ -905,17 +883,26 @@ async def _handler(
                         ),
                     )
                 )
-            git_clone = await init_containers.git_clone_container_v2(server)
+            git_providers = await self.nb_config.git_provider_helper.get_providers(user)
+            git_clone = await init_containers.git_clone_container_v2(
```

Comment:
I find it a good practice to provide argument names when calling a function from another module. It helps when reading the code and also avoid possible errors. Up to you to decide.

Reply:
Done.